ansible/playbooks/vars/set-custom-images.yaml

```diff
@@ -1,86 +1,239 @@
+# Configure your custom seldon deployment via ansible
```

Comment:
Should potentially a simple readme.md work better with few, ready to copy-paste and modify examples?

Reply:
I'll add that, would indeed be better.